lib/screens/search/category_episode.dart

```diff
@@ -0,0 +1,38 @@
+import 'package:flutter/material.dart';
+import 'package:riverpod/riverpod.dart';
+
+class CategoryEpisode extends StatelessWidget {
+  final double itemWidth;
+  double? imageHeight;
+  Map<String, String> episode;
```

Comment:
I'm using Map here because I'm still using sample data for this. Can I wait until its time to use the GraphQL apis or should I change it to Class now?